dmithamo/slickiest-blog-api
# #6 Configure Validators for Story Inputs

### What does this PR do?
This PR validates inout provided for creating a new story or updating one.

#### Description of Task to be completed
- check that title and body of story are not null
- use an `npm` package to generate a `slug` from each title.
- add `tags` as an attribute of a story

#### How should this be manually tested?

1. Run the following commands in Terminal

```bash
git clone git@github.com:dmithamo/slickiest-blog-api.git
cd slickiest-blog-api
git checkout ft-validate-story-input
yarn
yarn start:dev || yarn start:prod
```
2. Proceed to create a story at `http:localhost:3000` without a `title`, or a without a `body`, or with either of the two as empty strings.
The response return should indicate that errors were detected, and prevented the creation of a new story.
3. Creating a story with valid `title` and `body` should correctly create a new story.

#### What are the relevant Trello Cards?
N/A
#### Screenshots (if appropriate)

#### Anything else?

#### Checklist
